testblas/level1double.go

```diff
@@ -1086,6 +1086,12 @@ func DdotTest(t *testing.T, d Ddotter) {
 			t.Errorf("ddot: mismatch %v: expected %v, found %v", c.Name, c.DdotAns, dot)
 		}
 	}
+
```

Comment:
Incremented code doesn't depend on alignment, but I'll the test. Doesn't hurt to have more tests.